# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../arcload/wreck-abiflags.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- arcload-0.5/ecoff/wreckoff.c.vanilla	2020-08-13 20:39:40.001439263 +0000
+++ arcload-0.5/ecoff/wreckoff.c	2020-08-13 20:38:56.649439263 +0000
@@ -402,7 +402,7 @@
 		shnames[i] = (strtabs[ehdr.e_shstrndx] + shdr.sh_name);
 
 		if((shdr.sh_flags & SHF_ALLOC)) {
-			if(shdr.sh_type == SHT_REGINFO) {
+			if(shdr.sh_type == SHT_REGINFO || shdr.sh_type == 0x7000002A) {
 				fseek(f, shdr.sh_offset, SEEK_SET);
 				fread(&reginfo, sizeof(Elf32_RegInfo), 1, f);
 				swap_RegInfo(&reginfo);
